src/Item.php

```diff
@@ -117,9 +117,27 @@ public function get_item_tags(string $namespace, string $tag)
         return null;
     }
 
+    /**
+     * Get base URL of the item itself.
+     * Returns `<xml:base>` or feed base URL.
+     * Similar to `get_base()` but can safely be used during initialisation methods
+     * such as `get_links()` (`get_base()` and `get_links()` call each-other)
```

Comment:
If `xml:base` is not set, it will rely on the feed's `SimplePie::get_base()`, which does not depend on `Item::*`, so there should not be mutual dependencies anymore, unlike before this PR

Reply:
Ah, I missed that the comment is talking about `Item::get_base()`.

Reply:
Improved comment https://github.com/simplepie/simplepie/pull/861/commits/13a398d1b5d348e7113dedb3b5cc33063aa3c09a